src/commands/report.py

```diff
@@ -77,6 +77,11 @@ def validate_verbose_test_failure_reporting_ticket_limit(
     default="test-platform-results",
     type=click.STRING,
 )
+@click.option(
+    "--gcs-creds-file",
```

Comment:
Can you also update docs for this option, maybe like how to get this creds?

Reply:
1+ and since the client is set directly from the creds file, maybe specify the required format (or give an example ref).